sentry_sdk/integrations/__init__.py

```diff
@@ -9,11 +9,13 @@
 
 def _get_default_integrations():
     from .logging import LoggingIntegration
+    from .stdlib import StdlibIntegration
```

Comment:
probably meant to be httplib

Reply:
I wanted to make an stdlib integration with all the hooks against the stdlib instead of one integration per stdlib module

Reply:
ok, should we move the logging integration into this?

Reply:
I would keep that one separate because it's so special.

Reply:
Keep in mind though that if we put too much into one integration you can't enable/disable each of the "sub-integrations" one-by-one.

I don't understand why you do this though since the stdlibintegration is enabled by default anyway.